test/types/real/toBits.good

```diff
@@ -0,0 +1,4 @@
+param 4614256656552045848: uint(64) == 3.14159:real(64).toBits()
```

Comment:
I'm not so sure these numeric outputs are stable on different machines. It would make more sense IMO for the test to just check it can convert real to uint and back again and get the same value

Reply:
I considered that, but worried that that could potentially mask other issues (but maybe needlessly so as long as the destination isn't stored back into the originating variable).  I'd also been wondering how portable they will or will not be.  Are you worried about portability across our machines specifically (which would be easy to test manually on some and nightly on others) or portability if someone tries to run our tests elsewhere?

Reply:
I'm worried about portability across x86-64 vs 64-bit arm.